app/controllers/recipe_ingredients_controller.rb

```diff
@@ -0,0 +1,6 @@
+class RecipeIngredientsController < ApplicationController
+  def create
+    RecipeIngredient.create!(recipe: Recipe.find(params[:id]), ingredient: Ingredient.find(params[:add_ingredient]))
```

Comment:
Nice work setting up a joins controller! This is the most RESTful option for setting up the relationship between an ingredient and a recipe.